edbdeploy/data/templates/config.yml.j2

```diff
@@ -143,9 +141,7 @@ instances:
   public_ip:  {{ pooler_server['public_ip'] }}
   private_ip: {{ pooler_server['private_ip'] }}
   role:
-  - pgbouncer
```

Comment:
Is pgbouncer still installed and configured without this role?

Reply:
Yes. It is still installed by the `tpaexec` on pgbouncer nodes.